packages/block-editor/src/components/block-preview/auto.js

```diff
@@ -83,12 +93,23 @@ function ScaledBlockPreview( {
 					);
 					documentElement.style.position = 'absolute';
 					documentElement.style.width = '100%';
-					bodyElement.style.padding = __experimentalPadding + 'px';
+					documentElement.style.minHeight = '100%';
+					bodyElement.style.padding = padding;
 
 					// Necessary for contentResizeListener to work.
 					bodyElement.style.boxSizing = 'border-box';
 					bodyElement.style.position = 'absolute';
 					bodyElement.style.width = '100%';
+					bodyElement.style.minHeight = `${ __experimentalMinHeight }px`;
+
+					if ( __experimentalAlign || __experimentalJustify ) {
```

Comment:
Why do we need these styles? What are the use cases for `display:flex`?

Reply:
it is to align the block with in the preview container, both horizontally and vertically. Its is done conditionally no not effect `Block Inserter` preview.

<img width="181" alt="image" src="https://user-images.githubusercontent.com/1935113/203524517-dda88585-cdc1-444d-a41e-39c8c247b1d5.png">

Reply:
Can we do it with css only like we do for example in block patterns preview in patterns explorer? I'm not 100% sure it's the same thing we want, but it seems like it: 
<img width="1785" alt="Screenshot 2022-11-23 at 12 51 30 PM" src="https://user-images.githubusercontent.com/16275880/203528815-be49d6d4-d979-4ab3-926e-857499936c54.png">

Reply:
Container is not set 100% height here and alignment is taken care in parent. But in preview, full height is required and so flex alignment within preview panel.

<img width="1080" alt="image" src="https://user-images.githubusercontent.com/1935113/203974296-6c10d7d9-4f81-4d09-849d-5aa80ad7ce32.png">

Reverted the change from this PR and will take it to second iteration.